cli/lsp/jsr_resolver.rs

```diff
@@ -86,15 +62,62 @@ impl JsrResolver {
     specifier: &ModuleSpecifier,
   ) -> Option<ModuleSpecifier> {
     let req_ref = JsrPackageReqReference::from_str(specifier.as_str()).ok()?;
-    let nv = self.nv_by_req.get(req_ref.req())?;
-    let info = self.info_by_nv.get(nv)?;
+    let req = req_ref.req().clone();
+    let maybe_nv = self.nv_by_req.entry(req.clone()).or_insert_with(|| {
+      let name = req.name.clone();
+      let maybe_package_info = self
+        .info_by_name
+        .entry(name.clone())
+        .or_insert_with(|| read_cached_package_info(&name, &self.cache));
+      let package_info = maybe_package_info.as_ref()?;
+      let version = package_info
+        .versions
+        .keys()
+        .find(|v| req.version_req.matches(v))
```

Comment:
This hits the panic at https://github.com/denoland/deno_semver/blob/0.5.4/src/lib.rs#L265-L268 for incomplete specifiers like `jsr:@denotest/add@`. Gonna work around